.github/workflows/publish-java.yml

```diff
@@ -36,11 +41,21 @@ jobs:
             exit 1
```

Comment:
Please change the above branch check to `main`.

Reply:
Done.